docs/release-notes.rst

```diff
@@ -1,239 +1,210 @@
-- :func:`~scanpy.datasets.ebi_expression_atlas` allows convenient download of EBI expression atlas :noteversion:`1.4.1` :smaller:`thanks to I Virshup`
-- :func:`~scanpy.tl.marker_gene_overlap` computes overlaps of marker genes :noteversion:`1.4.1` :smaller:`thanks to M Luecken`
-- :func:`~scanpy.tl.filter_rank_genes_groups` filters out genes based on fold change and fraction of cells expressing genes :noteversion:`1.4.1` :smaller:`thanks to F Ramirez`
-- :func:`~scanpy.pp.normalize_total` replaces :func:`~scanpy.pp.normalize_per_cell`, is more efficient and provides a parameter to only normalize using a fraction of expressed genes :noteversion:`1.4.1` :smaller:`thanks to S Rybakov`
-- :func:`~scanpy.pp.downsample_counts` has been sped up, changed default value of `replace` parameter to `False` :noteversion:`1.4.1` :pr:`474` :smaller:`thanks to I Virshup`
-- :func:`~scanpy.pl.embedding_density` allows plots of cell densities on embeddings :noteversion:`1.4.1` :pr:`543` :smaller:`thanks to M Luecken`
-- :func:`~scanpy.external.tl.palantir` interfaces Palantir [Setty18]_ :noteversion:`1.4.1` :pr:`493` :smaller:`thanks to A Mousa`
+- Scanpy has a command line interface again. Invoking it with `scanpy somecommand [args]` calls `scanpy-somecommand [args]`, except for builtin commands (currently `scanpy settings`)  :pr:`604` :smaller:`P Angerer`
+- :func:`~scanpy.datasets.ebi_expression_atlas` allows convenient download of EBI expression atlas :smaller:`I Virshup`
+- :func:`~scanpy.tl.marker_gene_overlap` computes overlaps of marker genes :smaller:`M Luecken`
+- :func:`~scanpy.tl.filter_rank_genes_groups` filters out genes based on fold change and fraction of cells expressing genes :smaller:`F Ramirez`
+- :func:`~scanpy.pp.normalize_total` replaces :func:`~scanpy.pp.normalize_per_cell`, is more efficient and provides a parameter to only normalize using a fraction of expressed genes :smaller:`S Rybakov`
+- :func:`~scanpy.pp.downsample_counts` has been sped up, changed default value of `replace` parameter to `False`  :pr:`474` :smaller:`I Virshup`
+- :func:`~scanpy.pl.embedding_density` allows plots of cell densities on embeddings  :pr:`543` :smaller:`M Luecken`
+- :func:`~scanpy.external.tl.palantir` interfaces Palantir [Setty18]_  :pr:`493` :smaller:`A Mousa`
 
 Code design:
 
-- `.layers` support of scatter plots :noteversion:`1.4.1` :smaller:`thanks to F Ramirez`
-- fix double-logarithmization in compute of log fold change in :func:`~scanpy.tl.rank_genes_groups` :noteversion:`1.4.1` :smaller:`thanks to A Muñoz-Rojas`
-- fix return sections of docs :noteversion:`1.4.1` :smaller:`thanks to P Angerer`
+- `.layers` support of scatter plots :smaller:`F Ramirez`
+- fix double-logarithmization in compute of log fold change in :func:`~scanpy.tl.rank_genes_groups` :smaller:`A Muñoz-Rojas`
+- fix return sections of docs :smaller:`P Angerer`
 
 
-Version 1.3.*
--------------
+Version 1.3 :small:`2019-02-05`
```

Comment:
Does that mean the last 1.3.x release was then or 1.3.0?

In any case it would be clearer to attach dates to the individual subreleases, and leave the summarizing header without date.

If you want a date there, maybe do a date range? ``:small:`2018-xx-yy – 2019-02-05` ``

Reply:
Yes, yes... I agree... but we don't have the dates for the subversions.

I thought it became clear by adding the same date both to 1.4 and the latest of 1.4.*.

That's also not super beautiful but it gives this nice timeline in the sidebar on the left and makes the temporal development and progress very evident

![image](https://user-images.githubusercontent.com/16916678/71494759-d933dd80-2849-11ea-95fe-bad51cb655c1.png)

Giving the range would clutter this. Happy to remove the day and only maintain year and month on the section level, but have the day on the subsection level. Actually, I think that would look great and would make sense.

Reply:
OK, so since the class is carried over, you can just replace `:small:` with a `:date:` role and add a CSS rule that hides it in the main text while showing it in the sidebar.

Reply:
Yes, we can do this in the future! But the current solution works fine, too!